src/QuickMarcEditor/QuickMarcEditorContainer.js

```diff
@@ -8,7 +8,12 @@ import PropTypes from 'prop-types';
 import { withRouter } from 'react-router';
 import ReactRouterPropTypes from 'react-router-prop-types';
 
-import { stripesConnect } from '@folio/stripes/core';
+import {
+  stripesConnect,
```

Comment:
Please use `useStripes` in a functional component

Reply:
I use the already passed `stripes` property.